netket/sampler/rules/__init__.py

```diff
@@ -15,6 +15,7 @@
 from .local import LocalRule
 from .exchange import ExchangeRule
 from .hamiltonian import HamiltonianRule
+from .gaussian_continuous import GaussianRule
```

Comment:
can you rename it to `continouus_gaussian` so that they are packed together in alphabetical order?

Reply:
I would maybe call this just gaussian, I guess that is intrinsically continuous

Reply:
also if we want to be rigorous, ```GaussianRule``` should just go in ```.local``` I think!

Reply:
`local.py` is just a file. Those are not seen by the user.
I'd like to keep one Transition rule per file for now.
It will make it easier to split netket in sub packages if it ever happens in the future...